src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.CSharp.DesignTime.targets

```diff
@@ -42,25 +43,26 @@
     <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)ApplicationPropertyPage.CSharp.xaml">
       <Context>Project</Context>
     </PropertyPageSchema>
-    
+
     <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)BuildPropertyPage.CSharp.xaml">
       <Context>Project</Context>
     </PropertyPageSchema>
 
+    <!-- See: https://github.com/dotnet/project-system/pull/8235 -->
```

Comment:
I added this since I saw the commented code here and wondered why it was in the repo. We should always try to add information on why commented code exists in the code itself.